react/week2/todo-app/src/Todos/AddTodo.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+const AddTodo = ({ onAdd }) => {
```

Comment:
Maybe I would rather name this TodoInput or something like that, so it's not misunderstood with maybe a custom hook (that contains only reusable logic). You already (rightly) name your click event handler addTodo.